tiup/tiup-faq.md

```diff
@@ -0,0 +1,55 @@
+---
+title: TiUP FAQ
+category: tools
+---
+
+# TiUP FAQ
+
+## Can TiUP not use the official mirror source?
+
+TiUP supports specifying the mirror source through the `TIUP_MIRRORS` environment variable. The address of the mirror source can be a local directory or an HTTP server address. If your environment cannot access the network, you can create your own offline mirror source to use TiUP.
+
+## How do I put my own component into the TiUP mirrors?
+
+TiUP does not support third-party components for the time being, but the TiUP Team has developed the TiUP component development specifications and is developing the tiup-publish component. After everything is ready, a contributor can publish their own components to TiUP's official mirrors by using the `tiup publish <comp> <version>` command.
+
+## What is the difference between tiup-playground and tiup-cluster?
+
+The TiUP playground component is mainly used to build a stand-alone development environment on Linux or Mac operating systems. It helps you get started quickly and run a specified version of the TiUP cluster easily. The TiUP cluster component is mainly used to deploy and maintain a production environment cluster, which is usually a large-scale cluster.
+
+## How do I write the topology file for the tiup-cluster component?
+
+Refer to [these templates](https://github.com/pingcap/tiup/tree/master/examples) to write the topology file. The templates includes:
+
+1. Multi-DC deployment topology
+2. Minimal deployment topology
+3. Complete topology file
+
+You can edit your topology file based on the templates and your needs.
+
+## Can multiple instances be deployed on the same host?
+
+You can use the TiUP cluster component to deploy multiple instances on the same host, but with different ports and directories configured, otherwise directory and port conflicts might occur.
+
+## Are port and directory conflicts detected within the same cluster?
+
+Port and directory conflicts in the same cluster are detected during deployment and scaling. If there are any directory or port conflict, the deployment or scaling process is interrupted.
+
+## Are port and directory conflicts detected among different clusters?
+
+If multiple different clusters are deployed by the same TiUP control machine, the port and directory conflicts among these clusters are detected during deployment and scaling. If the clusters are deployed by different TiUP control machines, conflict detection is not supported currently.
```

Comment:
> If the clusters are deployed by different TiUP control machines, conflict detection is not supported currently.

I think `tiup cluster check` can detect this...
The trade-off is that users need to run this command by themselves.

Reply:
Same as https://github.com/pingcap/docs/pull/3052#discussion_r448154959. Can you provide a Chinese version first?And I‘ll align it in the English version. @lucklove 

FYI @lonng

Reply:
Confirmed with @lucklove, he will fix the deprecated content later in a new PR.